queries.sql

```diff
@@ -0,0 +1,134 @@
+PRAGMA foreign_keys = ON; 
+
+-- Tables
+CREATE TABLE organization (
+    id INTEGER PRIMARY KEY AUTOINCREMENT,
+    org_name VARCHAR(128) NOT NULL
+);
+CREATE TABLE channel (
+    id INTEGER PRIMARY KEY AUTOINCREMENT,
+    ch_name VARCHAR(128) NOT NULL,
+    organization_id INTEGER REFERENCES organization(id)
+);
+CREATE TABLE user (
+    id INTEGER PRIMARY KEY AUTOINCREMENT,
+    username VARCHAR(128) NOT NULL,
+    organization_id INTEGER REFERENCES organization(id)
+);
+CREATE TABLE message (
+    id INTEGER PRIMARY KEY AUTOINCREMENT,
+    dt DATETIME DEFAULT current_timestamp,
+    content VARCHAR(128) NOT NULL,
+    user_id INTEGER REFERENCES user(id),
+    recipient INTEGER REFERENCES user(id)
+);
+
+-- Join Tables
+CREATE TABLE channel_user (
+    channel_id INTEGER REFERENCES channel(id),
+    user_id INTEGER REFERENCES user(id)
+);
+CREATE TABLE channel_message (
+    channel_id INTEGER REFERENCES channel(id),
+    user_id INTEGER REFERENCES user(id),
+    dt DATETIME DEFAULT current_timestamp,
+    content VARCHAR(128) NOT NULL
+);
+
+INSERT INTO organization (org_name) VALUES ("Lambda School");
+
+INSERT INTO user (username,organization_id) VALUES ("Alice",1);
+INSERT INTO user (username,organization_id) VALUES ("Bob",1);
```

Comment:
Just a tip that You can also write multiple `INSERT` queries in one line by separating values with coma.